server/lib/format-barrier-data.js

```diff
@@ -12,8 +12,8 @@ module.exports.formatBarrierData = offer => {
 		formatted.secondaryPricing = copyTemplate.replace(PRICE_MATCH, secondaryPriceValue);
 	}
 
-	if (offer.pricingCopyPricePath && offer.pricingCopyTemplate) {
-		const pricingValue = get(offer, offer.pricingCopyPricePath);
+	const pricingValue = get(offer, offer.pricingCopyPricePath);
```

Comment:
I suspect similar issue at lines 22-23 and similar fix to it :)

Reply:
Changed this one as well now